docs/standard/exceptions/how-to-create-localized-exception-messages.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "How to: create user-defined exceptions with localized exception messages"
+description: "Learn how to create user-defined exceptions with localized exception messages"
+author: Youssef1313
+ms.author: ronpet
+ms.date: 09/02/2019
+---
+# How to: create user-defined exceptions with localized exception messages
+
+In this article, you will learn how to create user-defined exceptions that are inherited from the base <xref:System.Exception> class with localized exception messages using satellite assemblies.
+
+## Create custom exceptions
+
+.NET contains many different exceptions that you can use. However, in some cases when none of them meets your needs, you can create your own custom exceptions.
+
+Let's assume you want to create a `StudentNotFoundException` that contains a `StudentName` property.
+To create custom exceptions, Follow these steps:
+
+- Create a serializable class that inherits from <xref:System.Exception>. The class name should end in "Exception":
+
+    ```csharp
+    [Serializable]
+    public class StudentNotFoundException : Exception { }
+    ```
+
+- Add the default constructors:
+
+    ```csharp
+    [Serializable]
+    public class StudentNotFoundException : Exception
+    {
+        public StudentNotFoundException() { }
+
+        public StudentNotFoundException(string message)
+            : base(message) { }
+
+        public StudentNotFoundException(string message, Exception inner)
+            : base(message, inner) { }
+    }
+    ```
+
+- Add properties and more constructors, depending on your needs:
```

Comment:
Put the condition first, so the list item can be skipped fast by readers who don't need:
```suggestion
- If you need, define additional properties and constructors:
```